src/sparsezoo/analysis/helpers.py

```diff
@@ -12,27 +12,120 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import Tuple
+from typing import Dict, Optional, Tuple
 
 import numpy
 from onnx import ModelProto, NodeProto, numpy_helper
 
+from sparsezoo.analysis.onnx_helpers import (
+    NodeShape,
+    calculate_num_operations,
+    extract_node_id,
+    extract_node_shapes,
+    get_kernel_shape,
+    get_node_attributes,
+)
+
 
 __all__ = [
     "get_layer_and_op_counts",
     "get_node_four_block_sparsity",
     "get_node_num_four_block_zeros_and_size",
     "get_node_sparsity",
-    "get_layer_param",
+    "get_node_weight",
+    "get_node_bias",
     "get_node_num_zeros_and_size",
     "get_zero_point",
     "is_four_block_sparse_layer",
     "is_parameterized_prunable_layer",
     "is_quantized_layer",
     "is_sparse_layer",
+    "get_num_operations",
+    "extract_node_shapes",
 ]
 
 
+def _get_node_input(node, index, default=None):
```

Comment:
nit: "private" functions should go at the bottom of the file after exported ones